javascript/builtins/Intl.json

```diff
@@ -0,0 +1,1144 @@
+                  "version_added": null
+                },
+                "opera": {
+                  "version_added": "15"
+                },
+                "opera_android": {
+                  "version_added": null
+                },
+                "safari": {
+                  "version_added": "10"
+                },
+                "safari_ios": {
+                  "version_added": "10"
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "format": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat/format",
+              "support": {
+                "webview_android": {
+                  "version_added": false
+                },
+                "chrome": {
+                  "version_added": "24"
+                },
+                "chrome_android": {
+                  "version_added": "26"
+                },
+                "edge": {
+                  "version_added": true
+                },
+                "edge_mobile": {
+                  "version_added": true
+                },
+                "firefox": {
+                  "version_added": "29"
+                },
+                "firefox_android": {
+                  "version_added": "56"
+                },
+                "ie": {
+                  "version_added": "11"
+                },
+                "ie_mobile": {
+                  "version_added": false
+                },
+                "nodejs": {
+                  "version_added": null
+                },
+                "opera": {
+                  "version_added": "15"
+                },
+                "opera_android": {
+                  "version_added": null
+                },
+                "safari": {
+                  "version_added": "10"
+                },
+                "safari_ios": {
+                  "version_added": "10"
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "formatToParts": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat/formatToParts",
+              "support": {
+                "webview_android": {
+                  "version_added": false
+                },
+                "chrome": {
+                  "version_added": "55",
+                  "flag": {
+                    "type": "compile_flag",
+                    "name": "--datetime-format-to-parts"
+                  }
+                },
+                "chrome_android": {
+                  "version_added": false
```

Comment:
is it false or same as desktop?

Reply:
The Intl API generally mostly depends on the ICU library I think, so mobile and desktop versions might not always be the same (see Firefoxes), I'm setting this to `null`, as I don't know the answer here.